### ISSUE TITLE
Question: simple example of stopwordsauto

### ISSUE THREAD

**Issue by clukewatson** [julioraffo/matchit#2]
Howdy,

Thank you for the great package. I think the help files are very clear with one exception. I do not really understand what **stopwordsauto** is doing. I also do not understand what **swthreshold** does either. 

**Question:** could you give a practical example? 

**What I think / want it do be doing**: I have two columns of company names I want to match (simple bilateral matching with one dataset). Many companies have "LLC" or "CORP" or "CORPORATION", and these parts of the name string are essentially useless to me (they do not actually help identify matches). I believe that **stopwordsauto** is seeing that there are so many of these strings -- e.g., "LLC" and "CORP" -- and ignoring these strings. More specifically, it is as if **matchit** is deleting "LLC" and "COPR" from every string from which it is present _and then_ doing a standard **matchit** as if I did not specify **stopwordsauto.** The caveat being it might also be grabbing information that is useful; e.g., suppose there are many companies with USA in the title and USA is being removed or even just common letter groups such as "ING" or "TION". 

I do not even have a guess what **swthreshold** is doing...  Quote: _It sets the threshold of grams per observation to be included in the stopwords list.  Default is grams-per-observation = .2._ 

Thank you for your time,
Best,
Luke

**Comment by clukewatson** [comment]
Addendum: 

I guess I did not initially search enough. 

This [statalist forum link](https://www.statalist.org/forums/forum/general-stata-discussion/general/1040086-matchit-new-ado-to-merge-datasets-using-different-string-similarity-methods-i-e-fuzzy-match/page2) seems to show what the stopwords would be, and then one could set a threshold of say 0.5 and so only the number 1 result would be eliminated?

**Comment by julioraffoWIPO** [comment]
> Addendum: 
> 
> I guess I did not initially search enough. 
> 
> This [statalist forum link](https://www.statalist.org/forums/forum/general-stata-discussion/general/1040086-matchit-new-ado-to-merge-datasets-using-different-string-similarity-methods-i-e-fuzzy-match/page2) seems to show what the stopwords would be, and then one could set a threshold of say 0.5 and so only the number 1 result would be eliminated?

Correct. The best way to understand the option _stopw_ and its threshold is to run _matchit_ with the _diagnose_ option on. The third list contains the top frequent cross-grams in both files. The last column gives you the frequency that will be filtered by the threshold option.
